Pear/Cells/ChatTableViewCell.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  ChatTableViewCell.swift
+//  Pear
+//
+//  Created by Amy Chin Siu Huang on 5/9/21.
+//  Copyright © 2021 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+class ChatTableViewCell: UITableViewCell {
+
+    // MARK: - Private View Vars
+    private var chatBubble = UIView()
+    private var chatMessage = UILabel()
+    private var pairProfilePic = UIImageView()
+
+    // MARK: - Private Data Vars
+    static let reuseId = "chatReuseIdentifier"
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+        contentView.backgroundColor = .backgroundLightGreen
+
+        chatBubble.layer.cornerRadius = 16
+        chatBubble.backgroundColor = .red
```

Comment:
? is the chat bubble red when it hasn't sent yet

Reply:
oops I added this when I was debugging and forgot to remove haha